PLAN.md

```diff
@@ -0,0 +1,52 @@
+- Create own directory plugin?
+    - netrw not lightweight, pollutes clipboard history
+
+- Override window navigation commands
+    - File explorer window
+        - Fixed window to the left
+        - Window navigation
+            - http://blog.paulrugelhiatt.com/vim/2014/10/31/vim-tip-automatically-create-window-splits-with-movement.html
+            - Command: `oni.window.moveLeft`, `oni.window.moveRight`, `oni.window.moveUp`, `oni.window.moveDown`
+            - WindowManager - multiple Neovim editor instances
+
+- Window focus management
+    - Create Window Manager
+    - Have focus managed
+    - Pull keyboard out externally, have focused window handle input
+        - Keyboard focus for active window
+    - Create `deactivate`, `activate` concepts for editor
+    - Push `onKeyDown` to editor
+    - How to keep from 'drilling in' to file?
+
+- Rename scenario?
```

Comment:
This all sounds wonderful.
You say keep the keybindings from vim, which is a great idea.

And the mappings we could dynamically set the light weight windows to the mappings one would expect.
If I open a browser window, I want to use hjkl to navigate inside of them, if someone has mapped those to jkl;, then it should work for them as well, but do I expect other functionality like splits will open a new browser below?
To what extent will we handle vims mappings? Same with file explorers, it wouldn't be hard if you are saying use autocmds, just where do we draw the line?